schema/vrs.yaml

```diff
@@ -355,15 +355,6 @@ definitions:
           - $ref: "#/definitions/DerivedSequenceExpression"
       count:
         $ref: "#/definitions/Range"
-    example:
```

Comment:
Why did you remove the example?

Reply:
I plan to put this back once we have landed on a solution. This is more of a documentation concern and I do not want to have misleading examples get missed in the final presentation of the schema.